com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -220,20 +229,36 @@ internal NetworkObject CreateLocalNetworkObject(bool softCreate, uint prefabHash
                 }
                 else
                 {
-                    var prefabIndex = GetNetworkPrefabIndexOfHash(prefabHash);
+                    //See if there is a valid registered NetworkPrefabOverrideLink associated with the provided prefabHash
+                    GameObject networkPrefabReference = null;
+                    if(NetworkManager.NetworkConfig.NetworkPrefabOverrideLinks.ContainsKey(prefabHash))
+                    {
+                        switch (NetworkManager.NetworkConfig.NetworkPrefabOverrideLinks[prefabHash].Override)
+                        {
+                            default:
+                            case NetworkPrefabOverride.None:
+                                networkPrefabReference = NetworkManager.NetworkConfig.NetworkPrefabOverrideLinks[prefabHash].Prefab;
+                                break;
+                            case NetworkPrefabOverride.Hash:
+                            case NetworkPrefabOverride.Prefab:
+                                networkPrefabReference = NetworkManager.NetworkConfig.NetworkPrefabOverrideLinks[prefabHash].OverridingTargetPrefab;
+                                break;
+                        }
+                    }
 
-                    if (prefabIndex < 0)
+                    //If not, then there is an issue (user possibly didn't register the prefab properly?)
+                    if (networkPrefabReference == null)
                     {
                         if (NetworkLog.CurrentLogLevel <= LogLevel.Error)
                         {
-                            NetworkLog.LogError($"Failed to create object locally. [{nameof(prefabHash)}={prefabHash}]. Hash could not be found. Is the prefab registered?");
+                            NetworkLog.LogError($"Failed to create object locally. [{nameof(prefabHash)}={prefabHash}]. {nameof(NetworkPrefab)} could not be found. Is the prefab registered with {nameof(NetworkManager)}?");
```

Comment:
Similar to above, should we not exit when this happens?

Reply:
See the notes above for NetworkSceneManager.cs